pages/learner-onboarding/background-selection.js

```diff
@@ -1,12 +1,54 @@
 import LearnerOnboardingLayout from "../../components/LearnerOnboardingLayout";
+import Tts from "../../components/tts";
+import { colours } from "../../data/colours.js";
```

Comment:
Good shout on the curly bracket for the name, this got us to fix our path later for the avatar ✨